app/routes.py

```diff
@@ -1,2 +1,231 @@
-from flask import Blueprint
+from flask import Blueprint, request, make_response, jsonify
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import os
+from dotenv import load_dotenv
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods=["POST","GET"])
+def handle_tasks():
+    if request.method=="POST":
+        request_body = request.get_json()
+        if "completed_at" not in request_body.keys() or "description" not in request_body.keys() or "title" not in request_body.keys():
+            return make_response({"details": "Invalid data"}, 400) 
+        else:
+            new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
+            db.session.add(new_task)
+            db.session.commit()
+            return make_response({"task": {
+                    "id": new_task.task_id,
+                    "title": new_task.title,
+                    "description": new_task.description,
+                    "is_complete": False if new_task.completed_at is None else True
+                }}, 201)
```

Comment:
For tasks, you are repeating this code. You could create a helper function to reduce the number of times you are repeating this code. You could also move this code into your task model and create an instance method. You would call the instance method here.